testdata/dnn/download_models.py

```diff
@@ -612,10 +662,13 @@ def handle_bad_download(self):
                 sha='6d45d2f06150e9045631c7928093728b07c8b12d',
                 filename='onnx/data/output_resnet50_int8.pb'),
         ]),
-
+    # TODO: bad file
```

Comment:
Archive downloaded from GitHub has different checksum and contains different files, perhaps this model and related test should be reviewed separately.